src/ingress.pomerium.io_pomerium.yaml

```diff
@@ -14,262 +14,281 @@ spec:
+                    enum:
+                    - auth0
+                    - azure
+                    - google
+                    - okta
+                    - onelogin
+                    - oidc
+                    - ping
+                    - github
+                    type: string
+                  refreshDirectory:
+                    description: RefreshDirectory is no longer supported, please see
+                      https://docs.pomerium.com/docs/overview/upgrading#idp-directory-sync
+                    properties:
+                      interval:
+                        description: interval is the time that pomerium will sync
+                          your IDP directory.
+                        format: duration
                         type: string
-                      description: RequestParams see https://www.pomerium.com/reference/#identity-provider-request-params
-                      type: object
-                    requestParamsSecret:
-                      description:
-                        RequestParamsSecret is a reference to a secret for
-                        additional parameters you'd prefer not to provide in plaintext
-                      type: string
-                    scopes:
-                      description: Scopes see https://www.pomerium.com/reference/#identity-provider-scopes
-                      items:
+                      timeout:
+                        description: timeout is the maximum time allowed each run.
+                        format: duration
                         type: string
-                      type: array
-                    secret:
-                      description:
-                        Secret refers to a k8s secret containing IdP provider
-                        specific parameters and must contain at least `client_id` and
-                        `client_secret` map values
-                      minLength: 1
+                    required:
+                    - interval
+                    - timeout
+                    type: object
+                  requestParams:
+                    additionalProperties:
                       type: string
-                    url:
-                      description: URL is identity provider url, see https://www.pomerium.com/reference/#identity-provider-url
-                      format: uri
-                      pattern: ^https://
+                    description: RequestParams see https://www.pomerium.com/reference/#identity-provider-request-params
+                    type: object
+                  requestParamsSecret:
+                    description: RequestParamsSecret is a reference to a secret for
+                      additional parameters you'd prefer not to provide in plaintext.
+                    type: string
+                  scopes:
+                    description: Scopes see https://www.pomerium.com/reference/#identity-provider-scopes.
```

Comment:
the /reference/ links in lines 112 and 119 both redirect to the main reference page: https://www.pomerium.com/docs/reference

Correct RequestParams link: https://www.pomerium.com/docs/reference/identity-provider-request-params
Correct IdP Scopes link: https://www.pomerium.com/docs/reference/identity-provider-scopes